src/benchInsert.c

```diff
@@ -1570,6 +1570,69 @@ static int startMultiThreadInsertData(SDataBase* database, SSuperTable* stbInfo)
     return 0;
 }
 
+static int get_stb_inserted_rows(char* dbName, char* stbName, TAOS* taos) {
+    int rows = 0;
+    char command[SQL_BUFF_LEN];
+    sprintf(command, "select count(*) from %s.%s", dbName, stbName);
+    TAOS_RES* res = taos_query(taos, command);
+    int code = taos_errno(res);
+    if (code != 0) {
+        errorPrint(stderr, "Failed to execute <%s>, reason: %s\n", command, taos_errstr(res));
+        taos_free_result(res);
+        return -1;
+    }
+    TAOS_ROW row = taos_fetch_row(res);
+    if (row == NULL) {
+        rows = 0;
+    } else {
+        rows = (int)*(int64_t*)row[0];
+    }
+    taos_free_result(res);
+    return rows;
+}
+
+static void create_tsma(TSMA* tsma, SBenchConn* conn, char* stbName) {
+    char command[SQL_BUFF_LEN];
+    int len = snprintf(command, SQL_BUFF_LEN, 
+                       "create sma index %s on %s function(%s) interval (%s) sliding (%s)",
+                       tsma->name, stbName, tsma->func, tsma->interval, tsma->sliding);
+    if (tsma->custom) {
+        snprintf(command + len, SQL_BUFF_LEN - len, " %s", tsma->custom);
+    }
+    int code = queryDbExec(conn, command);
+    if (code == 0) {
+        infoPrint(stdout, "successfully create tsma with command <%s>\n", command);
+    }
+}
+
+static void* create_tsmas(void* args) {
+    tsmaThreadInfo* pThreadInfo = (tsmaThreadInfo*) args;
+    int inserted_rows = 0;
+    SBenchConn* conn = init_bench_conn();
```

Comment:
is it possible to get a NULL pointer here?

Reply:
yea, it could be, will fixed